docs/src/app/components/AppLeftNav.jsx

```diff
@@ -69,7 +69,20 @@ const AppLeftNav = React.createClass({
   },
 
   handleVersionChange: function(event, index, value) {
-    window.location = value;
+    if (value === this.firstNonPreReleaseVersion()) {
+      window.location = 'http://www.material-ui.com/';
```

Comment:
Out of curiosity, is there a reason why these links aren't relative paths? Just wondering if we ever wanted to check out gh-pages branch and test locally, these links would force navigate to whatever is running in production docs site.